dhall/tests/Dhall/Test/ETags.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Dhall.Test.ETags where
+
+import Data.Monoid ((<>))
+import Data.Text (Text)
+import Dhall.Util (Input(..))
+import Prelude hiding (FilePath)
+import Test.Tasty (TestTree)
+import Turtle (FilePath, format, fp)
+
+import qualified Data.Text        as Text
+import qualified Data.Text.IO     as Text.IO
+import qualified Dhall.ETags      as ETags
+import qualified Dhall.Test.Util  as Test.Util
+import qualified Test.Tasty       as Tasty
+import qualified Test.Tasty.HUnit as Tasty.HUnit
+import qualified Turtle
+
+etagsDirectory :: FilePath
+etagsDirectory = "./tests/etags"
+
+getTests :: IO TestTree
+getTests = do
+    etagsTests <- Test.Util.discover (Turtle.chars <* ".dhall") etagsTest (Turtle.lstree etagsDirectory)
+
+    let testTree = Tasty.testGroup "etags tests" [ etagsTests, etagsDirTest ]
+
+    return testTree
+
+etagsTest :: Text -> TestTree
+etagsTest prefix =
+    Tasty.HUnit.testCase (Text.unpack prefix) $ do
+        let inputFile  = Text.unpack (prefix <> ".dhall")
+        let outputFile = Text.unpack (prefix <> ".tags")
+
+        actualTags <- Test.Util.toDhallPath <$> ETags.generate (InputFile inputFile) [""] False
```

Comment:
`toDhallPath` is usually used for turning a file path into an import of that file. What does it do here?

Reply:
Windows generates paths to files with `\`, and in tests I generated with '/'. So yes, it should 'normalize' paths.

Reply:
Ah ok. `toDhallPath` isn't the right name in this context. Maybe duplicate the definition and call it `fixPaths` or so.

Reply:
Done.